gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -396,8 +474,17 @@ service Gateway {
           - the process is invalid (e.g. an event-based gateway has an outgoing sequence flow to a task)
    */
   rpc DeployProcess (DeployProcessRequest) returns (DeployProcessResponse) {
+    // deprecated, replaced by DeployResource rpc
+    option deprecated = true;
   }
 
+  /*
+    Deploys one or more resources (i.e. processes or decision models) to Zeebe.
+  */
+  rpc DeployResource (DeployRequest) returns (DeployResponse) {
+  }
```

Comment:
I'm a bit unsure about the naming.

❓ @remcowesterhoud Should it be `DeployResource` or `Deploy`
❓ @remcowesterhoud Should it be `DeployResourceRequest` or `DeployRequest`
❓ @remcowesterhoud Should it be `DeployResourceResponse` or `DeployResponse`

Reply:
I personally like `DeployResource` over just `Deploy`. I don't have a very strong preference, but this gives a small amount of context about what you are deploying.

Reply:
And then also `DeployResourceRequest` and `DeployResourceResponse`. I think it's a good practice to keep those names aligned.

Reply:
Thanks, I like it 👍